.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
В репозитории у этого файла нет флага "выполнимый", поэтому он сработает только на Windows, но на Linux/*nix будет ошибка:
```
# git commit
hint: The '.husky/pre-commit' hook was ignored because it's not set as executable.
hint: You can disable this warning with `git config advice.ignoredHook false`.
```

Такое бывает, если инициализировать Husky из-под Windows.

Для исправления нужно добавить файлу pre-commit флаг выполнимого:
```
diff --git a/.husky/pre-commit b/.husky/pre-commit
old mode 100644
new mode 100755
```

Reply:
Исправлено